pkg/loader_test.go

```diff
@@ -90,7 +90,7 @@ func TestFileLoader_Error(t *testing.T) {
 			name: "path escapes parent",
 			url: mustParseURL(testutil.PerGOOS{
 				Default: "file:///file/that/does/not/exist",
-				Windows: "file://c:/file/that/does/not/exist",
+				Windows: "file://C:/file/that/does/not/exist",
```

Comment:
To fix this test we could maybe use [filepath.VolumeName](https://pkg.go.dev/path/filepath#VolumeName) on os.Getwd (just here in the test) to figure out which drive to base the tests on, instead of hardcoding `C:`

Reply:
Alright. Since none of us are developing on windows then I agree to keep the tests a little tailored to GitHubs runners